glossary.qmd

```diff
@@ -0,0 +1,353 @@
+
+A file format for tabular data with [internal chunking](#chunk) and [internal compression](#internal-compression).
+
+#### PDAL
+
+The _Point Data Abstraction Library_, a widely-used open-source library for converting between different [point cloud data](#point-cloud-data) formats and managing point cloud data.
+
+#### PMTiles
+
+A type of [cloud-optimized](#cloud-optimized) [archive format](#archive-format) for tiled data. It can be used with either [vector](#vector-data) or [raster](#raster-data) tiled data, but is most often used with [Mapbox Vector Tile](#mapbox-vector-tile) files. The individual tiles in a PMTiles file are accessible via [HTTP range requests](#http-range-request).
+
+#### Point Cloud Data
+
+A type of geospatial data storing three dimensional point locations along with attributes for each point. Point cloud data may come from LIDAR sensors or other photogrammetry, and may represent three-dimensional terrain or buildings.
+
+#### PROJJSON
+
+A [projection definition](#coordinate-reference-system-crs) that uses JSON for encoding.
+
+The specification is [defined as part of the PROJ project](https://proj.org/en/9.3/specifications/projjson.html) and used as part of the [GeoParquet](#geoparquet) [vector file](#vector-data) format.
+
+#### Random access
+
+The ability to quickly fetch _part_ of a file without reading the entire file.
+
+For example, consider videos on YouTube. If you select to watch a video starting from the ten minute mark, YouTube does not need to download the video up to that point. Rather it is able to use the [metadata](#metadata) from the video file to know what byte range in the video file corresponds to the ten minute mark, and then use [HTTP range requests](#http-range-request) to download only the part of the video you've reqested.
+
+The ability to perform efficient random access over a network is a core part of what makes data [cloud-optimized](#cloud-optimized).
+
+#### Raster data
+
+A type of geospatial data that stores regularly-gridded data. This often comes from aerial or satellite imagery sensors.
+
+#### Shapefile
+
+A [vector data](#vector-data) file format. There are [many reasons](http://switchfromshapefile.org/) to no longer use Shapefile.
+
+#### Space-filling curve
+
+An algorithm that translates two- or n- dimensional data into one-dimensional data. In practice, this is used as part of [spatial indexes](#spatial-index) to group [vector geometries](#vector-data) nearby within a file according to their two-dimensional location.
+
+#### Spatial index
+
+A data structure used for searching through spatial data more efficiently.
+
+For further reading: [A dive into spatial search algorithms](https://blog.mapbox.com/a-dive-into-spatial-search-algorithms-ebd0c5e39d2a).
+
+#### Tagged Image File Format (TIFF)
+
+A file format for image data that supports [lossless compression](#lossless-compression).
+
+#### Vector data
+
+A type of geospatial data to represent points, lines, and polygons.
```

Comment:
and/or ?

Reply:
I was suggesting `and/or` some vector formats can't mix in the same files - many tools don't support mixed geometry, etc... probably irrelevant to this context `and` is the most common way to write the sentence.